hivemq-edge/src/main/java/com/hivemq/mqtt/services/PublishDistributorImpl.java

```diff
@@ -218,6 +221,37 @@ private ListenableFuture<PublishStatus> handlePublish(
                 clientSession.getQueueLimit());
     }
 
+    private @NotNull SettableFuture<PublishStatus> handlePublishForBridgeForwarder(
+            final @NotNull PUBLISH publish,
+            final @NotNull String client,
+            boolean retainAsPublished,
+            final @NotNull ImmutableIntArray subscriptionIdentifier,
+            final @NotNull Long queueLimit,
+            int appliedQoS) {
+        // update with the configuration of the bridge, if it is a bridge client
+        final CustomBridgeLimitations customBridgeLimitations = getBridgeConfig(client);
+        long appliedQueueLimit = queueLimit;
+
+        if (customBridgeLimitations != null) {
+            final Long queueLimitFromConfig = customBridgeLimitations.queueLimit;
+            if (queueLimitFromConfig != null) {
+                // bridges can overwrite the default
```

Comment:
I would log something like this on info.

Reply:
no we shouldnt do this. This would scale the log messages linear with the amount of messages. Logging even competes for exclusive access => contention.